src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -91,6 +91,14 @@ public static class DefaultMessageHeaders
         /// </summary>
         public const string ContentType = "content-type";
 
+        /// <summary>
+        ///     The encryption Key Identifier.
```

Comment:
Why you always write `Key Identifier` and `Key Rotation` with capitalized first letter?

Reply:
Ouch my bad 😲 .... maybe is due to German 🤣